likecoin/admin/metabox.php

```diff
@@ -79,14 +79,27 @@ function likecoin_parse_publish_status( $publish_params ) {
  */
 function likecoin_parse_iscn_status( $publish_params ) {
 	global $post;
-	$post_id               = $post->ID;
-	$result                = array();
-	$iscn_hash             = $publish_params['iscn_hash'];
+	$post_id                      = $post->ID;
+	$result                       = array();
+	$iscn_testnet_info            = get_post_meta( $post_id, LC_ISCN_DEV_INFO, true );
+	$iscn_mainnet_info            = get_post_meta( $post_id, LC_ISCN_INFO, true );
+	$iscn_hash                    = $publish_params['iscn_hash'];
+	$stargate_data_api_endpoint   = null;
+	$stargate_static_api_endpoint = null;
+	if ( $iscn_testnet_info || $iscn_mainnet_info ) {
```

Comment:
Would using if else if be clearer here?
`if ($iscn_testnet_info) else if ($iscn_mainnet_info)`